docs/markdown/guides/testing/docs-alert/docs-alert.md

```diff
@@ -8,10 +9,12 @@ QUESTION: Use Question to pose a question to the reader, kind of like a mini-qui
 
 Summary: Use Summary to provide a two- or three-sentence synopsis of the page or section content, so readers can figure out whether this is the right place for them.
 
-TLDR: Use TLDR if you can provide the essential information about a page or section in a sentence or two. 
+TLDR: Use TLDR if you can provide the essential information about a page or section in a sentence or two.
 
 CRITICAL: Use Critical to call out potential bad stuff or alert the reader they ought to be careful before doing something. For example, Warning: Running `rm` with the `-f` option will delete write-protected files or directories without prompting you.
 
 IMPORTANT: Use Important for information that's crucial to comprehending the text or to completing some task.
 
 HELPFUL: Use Best practice to call out practices that are known to be successful or better than alternatives.
+
+Something else
```

Comment:
Do we need this line still?

Reply:
Actually not anymore, there was an issue with the linter if we had 2 empty lines at the end of the file. 

Fixed by adding support for alert blocks at the end of a markdown file.